src/FakeItEasy/FakeOptionsExtensions.cs

```diff
@@ -16,13 +16,27 @@ public static class FakeOptionsExtensions
         /// <param name="options">Options used to create the fake object.</param>
         /// <returns>An options object.</returns>
         public static IFakeOptions<T> Strict<T>(this IFakeOptions<T> options) where T : class
+        {
+            return options.Strict(StrictFakeOptions.None);
+        }
+
+        /// <summary>
+        /// Makes the fake strict. This means that any call to the fake
+        /// that has not been explicitly configured will throw an exception,
+        /// except calls to the System.Object methods specified in <c>options</c>.

```

Comment:
```suggestion
        /// except calls to the System.Object methods specified in <c>strictOptions</c>.
```

Alternatively, a `paramref``? I'm not fussy.

Reply:
Oh! I didn't know about paramref. I'll look into it